packages/openapi-ts/src/openApi/v3/parser/getModelProperties.ts

```diff
@@ -118,7 +118,10 @@ export const getModelProperties = ({
         description: property.description || null,
         exclusiveMaximum: property.exclusiveMaximum,
         exclusiveMinimum: property.exclusiveMinimum,
-        format: property.format,
+        format:
+          property.type === 'array'
+            ? property.items?.format ?? property.format
+            : property.format,
         in: '',
```

Comment:
I'm not 100% sure if this is compliant, but it seems to break nothing and may have been a long standing oversight.

If you have `Array<string format:date-time>` in the OpenAPI doc right now, then the format doesn't propagate to the model because it's assumed there's a $ref in play for arrays, when actually arrays could have a items objective with type/format on